static/app/views/starfish/views/appStartup/screenSummary/startTypeSelector.tsx

```diff
@@ -15,15 +16,29 @@ export function StartTypeSelector() {
 
   const value = decodeScalar(location.query[SpanMetricsField.APP_START_TYPE]) ?? '';
 
+  useEffect(() => {
+    if (!value) {
+      browserHistory.replace({
+        ...location,
+        query: {
+          ...location.query,
+          [SpanMetricsField.APP_START_TYPE]: COLD_START_TYPE,
+        },
+      });
+    }
+
+    // This hook should only run once to set the default type
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🤔 what if someone manually removes the start type? Would it be safer to add `value` as a dep, remove the eslint disable, and run the hook whenever the value is missing, rather than just on the first run?

Reply:
Aside: it's a little risky for simpler components like selectors to do this kind of redirecting on mount, it can cause wacky URL thrashing for one thing. e.g., if this component mounts late, after a lot of the page already rendered, it'll cause a redirect and the screen will flash. It's a lot tidier to do this kind of thing somewhere higher and closer to the routing. Maybe the screen summary component?

Reply:
Great points! I'll address your comments in another PR. I just have a few chained PRs and they make some changes that would conflict with your suggestions so I'll get it in after those happen. Thanks George